package.json

```diff
@@ -34,6 +35,7 @@
         "eslint-plugin-vue": "^9.2.0",
         "husky": "^8.0.1",
         "jest": "^29.1.2",
+        "jest-environment-jsdom": "^29.2.1",
```

Comment:
Can we move these package.json changes into a separate pull request? 
Generally, it's better to tie to the actually dependencies changed that rely on it. Also, we should review what they're used for.

Reply:
@RageZBla redid the commits to touch only the GH workflows